### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by ThanhNghiTang**

### CODE REVIEW

File: shared/components/page-header/__stories__/index.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import PageHeader from '../PageHeader';
+
+const count = [].length;
```
Comment (RobbinHabermehl) [inline shared/components/page-header/__stories__/index.jsx]: This doesn't really do anything, it just resolves to `0`.

File: shared/components/page-header/__stories__/index.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+storiesOf('PageHeader', module)
+  .addDecorator(withInfo)
+  .add(
+    'Time Entries',
+    () => (
+      <PageHeader
+        title="Timesheets"
+        subtitle={`${count} ${count === 1 ? 'Entry' : 'Entries'}`}
```
Comment (RobbinHabermehl) [inline shared/components/page-header/__stories__/index.jsx]: Having count statically at `0` it doesn't make sense to add this if-statement here.

**Review by RobbinHabermehl — CHANGES_REQUESTED** [review]

File: shared/components/page-header/__stories__/index.jsx (hunk 1 above)
Comment (ThanhNghiTang) [reply shared/components/page-header/__stories__/index.jsx]: What does the story have to display? The exact amount of items in an array of time entries/clients/ team members? Or do i just use mock data?

Comment (RobbinHabermehl) [reply shared/components/page-header/__stories__/index.jsx]: Considering it's a pretty basic component I would use a static title.

Comment (ThanhNghiTang) [reply shared/components/page-header/__stories__/index.jsx]:
```
const arrayOfItems = ['Item 1', 'Item 2', 'Item 3'];
const count = arrayOfItems.length;
```

Is this a suitable fix?

Comment (RobbinHabermehl) [reply shared/components/page-header/__stories__/index.jsx]: That's not what I meant, I would just do `subtitle="2 entries"`. There's no use for defining an array and then counting the length. It's the same as writing `count = 3` in this case.

Comment (ThanhNghiTang) [reply shared/components/page-header/__stories__/index.jsx]: Ok, done!

**Review by RobbinHabermehl — APPROVED** [review]

**Review by RobbinHabermehl — APPROVED** [review]

**Pull request merged**